framework/src/application/node/transport/transport.js

```diff
@@ -71,7 +71,7 @@ class Transport {
 
 	handleBroadcastTransaction(transaction) {
 		this.broadcaster.enqueueTransactionId(transaction.id);
-		this.channel.publish('app:transactions:change', transaction.toJSON());
+		this.channel.publish('app:transactions:change', transaction);
```

Comment:
```suggestion
		this.channel.publish('app:transactions:change', transactionto.JSON());
```
this is class instance so it should be serialized